### PR TITLE
Remove mentions of compatibility.js from examples

### PR DESCRIPTION
**Pull request by wojtekmaj**

### CODE REVIEW

File: docs/contents/getting_started/index.md [hunk 1]
```diff
@@ -68,7 +68,6 @@ $ cd pdf.js
 │   └── pdf.worker.js                      - core layer
 └── web/
     ├── cmaps/                             - character maps(required by core)
-    ├── compatibility.js                   - polyfills for missing features
```
Comment (Snuffleupagus) [inline docs/contents/getting_started/index.md]: This was removed in PR #8819; please rebase your patch to the *current* `master`.

Comment (wojtekmaj) [reply docs/contents/getting_started/index.md]: Done, sorry for the wait

Comment (Snuffleupagus) [reply docs/contents/getting_started/index.md]: We don't allow merge commits in PRs, please squash this into *one commit*; see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits for additional information.

Comment (wojtekmaj) [reply docs/contents/getting_started/index.md]: You taught me something today! ;) Done. Thanks!

**Review by Snuffleupagus — APPROVED** [review]
Thanks for the patch!

**Pull request merged**